src/config/dynamic-theme-fixes.config

```diff
@@ -19666,7 +19666,7 @@ span[class*="nk-icon_id_business-feedback-task"] > svg > g > path
 
 ================================
 
-na.myconnectwise.net
+*.myconnectwise.net
```

Comment:
Sitefixes must be sorted alphabetically.
When you remove main URL and change it to another then the fix must be moved - in this case before `*.myworkday.com`